### ISSUE TITLE
trained model error

### ISSUE THREAD

**Issue by taylorlu** [mravanelli/SincNet#32]
Hi, the [trained model](https://bitbucket.org/mravanelli/sincnet_models/) seems not work when **compute_d_vector**:
`Missing key(s) in state_dict: "conv.0.low_hz_", "conv.0.band_hz_"`
the keys in checkpoint_load['CNN_model_par']:
```
conv.0.filt_b1
conv.0.filt_band
conv.1.weight
conv.1.bias
conv.2.weight
conv.2.bias
bn.0.weight
bn.0.bias
bn.0.running_mean
bn.0.running_var
bn.1.weight
bn.1.bias
bn.1.running_mean
bn.1.running_var
bn.2.weight
bn.2.bias
bn.2.running_mean
bn.2.running_var
ln.0.gamma
ln.0.beta
ln.1.gamma
ln.1.beta
ln.2.gamma
ln.2.beta
ln0.gamma
ln0.beta
```
Thanks a lot.

**Comment by mravanelli** [comment]
OK, thank you for pointing it. It seems a model computed with the previous
version of SincNet. Let me update it as soon as possible!

Mirco

**Comment by mravanelli** [comment]
Hi,
could you try with the new model in the external repository and with the new compute_d_vector code?

**Comment by taylorlu** [comment]
@mravanelli
Thanks very much, it works well now, the d-vector is in 2048 dim.
For open-set speaker verification, I think the model can use a more discriminative loss function such as arcface/amsoftmax insteads of softmax on the top of CNN/DNN layers.

**Comment by mravanelli** [comment]
Sure, you can try it!
